theano/gof/vm.py

```diff
@@ -179,6 +186,13 @@ def __call__(self):
                     t1 = time.time()
                     self.call_counts[i] += 1
                     self.call_times[i] += t1 - t0
+                    for ins in node.inputs:
+                        info = self.reallocated_info.get(ins, None)
+                        if info:
+                            for outs in info[1]:
+                                if not outs[0]:
```

Comment:
@nouiz 
I think if this outs is allocated, then we skip it, until find an outs which havent been allocated. Is override available here?